test/unit/org/apache/cassandra/cql3/CustomNowInSecondsTest.java

```diff
@@ -73,6 +79,48 @@ private void testSelectQuery(boolean prepared)
         assertEquals(0, executeSelect(format("SELECT * FROM %s.%s", ks, tbl), now + day + 1, prepared).size());
     }
 
+    @Test
+    public void testSelectQueryWitInthOverflow()
+    {
+        testSelectQueryOverflowingIntTimestamps(false);
+        testSelectQueryOverflowingIntTimestamps(true);
+    }
+
+    private void testSelectQueryOverflowingIntTimestamps(boolean prepared)
```

Comment:
These two methods could have the same name.

Reply:
Done